src/routes/search-product-name.router.js

```diff
@@ -0,0 +1,8 @@
+const express = require('express');
+const SearchProductController = require('../controllers/search-product-name.controller');
```

Comment:
```suggestion
const searchProductController = require('../controllers/search-product-name.controller');
```
자바스크립트에서는 class naming이 아닌경우 대문자로 시작하는 변수명을 짓지 않는것이 일반적입니다!